src/renderer/components/InsecureUserProfile.tsx

```
@@ -0,0 +1,91 @@
+  const renderUserBio = () => {
+    return <div dangerouslySetInnerHTML={{ __html: userData?.bio }} />;
+  };
+  
+  // ISSUE: Direct DOM manipulation with user input
+  const updateTitle = (title: string) => {
+    document.getElementById('user-title')!.innerHTML = title;
+  };
+  
+  // ISSUE: eval usage
+  const executeUserScript = (script: string) => {
+    eval(script);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)